crates/bevy_render/src/camera/projection.rs

```diff
@@ -154,17 +156,49 @@ pub enum ScalingMode {
     FixedHorizontal(f32),
 }
 
+/// A camera using `OrthographicProjection` displays the world within a rectangular prism (the "viewport", the "frustum", or
+/// "field of view"). Objects that do not lie in this prism are culled ("frustum culling"): in other words, they are not displayed.
+/// The size of the field of view is specified in "world units".
+///
+/// The camera displays objects in its field of view within a 2D window on the computer screen. If the window is smaller
+/// than the field of view, then some objects will naturally not appear on the screen. The relationship between the field of view
+/// and the window is governed by [`window_origin`].
```

Comment:
That last sentence feels a bit off to me - aside from it being a bit vague, I think the purpose of window_origin should be explained within the struct or a related function, not on in the general overview.